<ui version="4.0" >
 <class>build</class>
 <widget class="QWidget" name="build" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>169</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="KTabWidget" name="ktabwidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="errs" >
      <attribute name="title" >
       <string>Errors and Warnings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QTreeWidget" name="errTreeWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string comment="Header for the file name collumn">File</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string comment="Header for the line number collumn">Line</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string comment="Header for the error message collumn">Message</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="output" >
      <attribute name="title" >
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled" >
          <bool>false</bool>
         </property>
         <property name="lineWrapMode" >
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settings" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Working Directory:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>buildDir</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Build Command:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>buildCmd</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3" >
        <widget class="KLineEdit" name="buildCmd" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Clean Command:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>cleanCmd</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="KLineEdit" name="cleanCmd" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Quick Compile:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>quickComp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3" >
        <widget class="KLineEdit" name="quickComp" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <widget class="KLineEdit" name="buildDir" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="KPushButton" name="browse" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
